jupyterlab_launcher/handlers.py

```diff
@@ -176,17 +176,27 @@ def render_template(self, name, **ns):
 def add_handlers(web_app, config):
     """Add the appropriate handlers to the web app.
     """
+    base_url = web_app.settings['base_url']
+
     # Normalize directories.
     for name in config.trait_names():
         if not name.endswith('_dir'):
             continue
         value = getattr(config, name)
         setattr(config, name, value.replace(os.sep, '/'))
 
+    # Normalize urls.
+    for name in config.trait_names():
+        if not name.endswith('_url'):
+            continue
```

Comment:
What if someone was to add a config option `npm_registry_url` at some point? Wouldn't this test be too general?

Reply:
Good point, we should only handle urls that start with `/`.